// ramda/es/ramda_index.js
export { default as F } from ".../src/F.js";
export { default as T } from ".../src/T.js";
export { default as __ } from ".../src/__.js";
export { default as add } from ".../src/add.js";
export { default as addIndex } from ".../src/addIndex.js";
export { default as adjust } from ".../src/adjust.js";
export { default as all } from ".../src/all.js";
export { default as allPass } from ".../src/allPass.js";
export { default as always } from ".../src/always.js";
export { default as and } from ".../src/and.js";
export { default as any } from ".../src/any.js";
export { default as anyPass } from ".../src/anyPass.js";
export { default as ap } from ".../src/ap.js";
export { default as aperture } from ".../src/aperture.js";
export { default as append } from ".../src/append.js";
export { default as apply } from ".../src/apply.js";
export { default as applySpec } from ".../src/applySpec.js";
export { default as applyTo } from ".../src/applyTo.js";
export { default as ascend } from ".../src/ascend.js";
export { default as assoc } from ".../src/assoc.js";
export { default as assocPath } from ".../src/assocPath.js";
export { default as binary } from ".../src/binary.js";
export { default as bind } from ".../src/bind.js";
export { default as both } from ".../src/both.js";
export { default as call } from ".../src/call.js";
export { default as chain } from ".../src/chain.js";
export { default as clamp } from ".../src/clamp.js";
export { default as clone } from ".../src/clone.js";
export { default as collectBy } from ".../src/collectBy.js";
export { default as comparator } from ".../src/comparator.js";
export { default as complement } from ".../src/complement.js";
export { default as compose } from ".../src/compose.js";
export { default as composeWith } from ".../src/composeWith.js";
export { default as concat } from ".../src/concat.js";
export { default as cond } from ".../src/cond.js";
export { default as construct } from ".../src/construct.js";
export { default as constructN } from ".../src/constructN.js";
export { default as converge } from ".../src/converge.js";
export { default as count } from ".../src/count.js";
export { default as countBy } from ".../src/countBy.js";
export { default as curry } from ".../src/curry.js";
export { default as curryN } from ".../src/curryN.js";
export { default as dec } from ".../src/dec.js";
export { default as defaultTo } from ".../src/defaultTo.js";
export { default as descend } from ".../src/descend.js";
export { default as difference } from ".../src/difference.js";
export { default as differenceWith } from ".../src/differenceWith.js";
export { default as dissoc } from ".../src/dissoc.js";
export { default as dissocPath } from ".../src/dissocPath.js";
export { default as divide } from ".../src/divide.js";
export { default as drop } from ".../src/drop.js";
export { default as dropLast } from ".../src/dropLast.js";
export { default as dropLastWhile } from ".../src/dropLastWhile.js";
export { default as dropRepeats } from ".../src/dropRepeats.js";
export { default as dropRepeatsWith } from ".../src/dropRepeatsWith.js";
export { default as dropWhile } from ".../src/dropWhile.js";
export { default as either } from ".../src/either.js";
export { default as empty } from ".../src/empty.js";
export { default as endsWith } from ".../src/endsWith.js";
export { default as eqBy } from ".../src/eqBy.js";
export { default as eqProps } from ".../src/eqProps.js";
export { default as equals } from ".../src/equals.js";
export { default as evolve } from ".../src/evolve.js";
export { default as filter } from ".../src/filter.js";
export { default as find } from ".../src/find.js";
export { default as findIndex } from ".../src/findIndex.js";
export { default as findLast } from ".../src/findLast.js";
export { default as findLastIndex } from ".../src/findLastIndex.js";
export { default as flatten } from ".../src/flatten.js";
export { default as flip } from ".../src/flip.js";
export { default as forEach } from ".../src/forEach.js";
export { default as forEachObjIndexed } from ".../src/forEachObjIndexed.js";
export { default as fromPairs } from ".../src/fromPairs.js";
export { default as groupBy } from ".../src/groupBy.js";
export { default as groupWith } from ".../src/groupWith.js";
export { default as gt } from ".../src/gt.js";
export { default as gte } from ".../src/gte.js";
export { default as has } from ".../src/has.js";
export { default as hasIn } from ".../src/hasIn.js";
export { default as hasPath } from ".../src/hasPath.js";
export { default as head } from ".../src/head.js";
export { default as identical } from ".../src/identical.js";
export { default as identity } from ".../src/identity.js";
export { default as ifElse } from ".../src/ifElse.js";
export { default as inc } from ".../src/inc.js";
export { default as includes } from ".../src/includes.js";
export { default as indexBy } from ".../src/indexBy.js";
export { default as indexOf } from ".../src/indexOf.js";
export { default as init } from ".../src/init.js";
export { default as innerJoin } from ".../src/innerJoin.js";
export { default as insert } from ".../src/insert.js";
export { default as insertAll } from ".../src/insertAll.js";
export { default as intersection } from ".../src/intersection.js";
export { default as intersperse } from ".../src/intersperse.js";
export { default as into } from ".../src/into.js";
export { default as invert } from ".../src/invert.js";
export { default as invertObj } from ".../src/invertObj.js";
export { default as invoker } from ".../src/invoker.js";
export { default as is } from ".../src/is.js";
export { default as isEmpty } from ".../src/isEmpty.js";
export { default as isNil } from ".../src/isNil.js";
export { default as join } from ".../src/join.js";
export { default as juxt } from ".../src/juxt.js";
export { default as keys } from ".../src/keys.js";
export { default as keysIn } from ".../src/keysIn.js";
export { default as last } from ".../src/last.js";
export { default as lastIndexOf } from ".../src/lastIndexOf.js";
export { default as length } from ".../src/length.js";
export { default as lens } from ".../src/lens.js";
export { default as lensIndex } from ".../src/lensIndex.js";
export { default as lensPath } from ".../src/lensPath.js";
export { default as lensProp } from ".../src/lensProp.js";
export { default as lift } from ".../src/lift.js";
export { default as liftN } from ".../src/liftN.js";
export { default as lt } from ".../src/lt.js";
export { default as lte } from ".../src/lte.js";
export { default as map } from ".../src/map.js";
export { default as mapAccum } from ".../src/mapAccum.js";
export { default as mapAccumRight } from ".../src/mapAccumRight.js";
export { default as mapObjIndexed } from ".../src/mapObjIndexed.js";
export { default as match } from ".../src/match.js";
export { default as mathMod } from ".../src/mathMod.js";
export { default as max } from ".../src/max.js";
export { default as maxBy } from ".../src/maxBy.js";
export { default as mean } from ".../src/mean.js";
export { default as median } from ".../src/median.js";
export { default as memoizeWith } from ".../src/memoizeWith.js";
export { default as mergeAll } from ".../src/mergeAll.js";
export { default as mergeDeepLeft } from ".../src/mergeDeepLeft.js";
export { default as mergeDeepRight } from ".../src/mergeDeepRight.js";
export { default as mergeDeepWith } from ".../src/mergeDeepWith.js";
export { default as mergeDeepWithKey } from ".../src/mergeDeepWithKey.js";
export { default as mergeLeft } from ".../src/mergeLeft.js";
export { default as mergeRight } from ".../src/mergeRight.js";
export { default as mergeWith } from ".../src/mergeWith.js";
export { default as mergeWithKey } from ".../src/mergeWithKey.js";
export { default as min } from ".../src/min.js";
export { default as minBy } from ".../src/minBy.js";
export { default as modify } from ".../src/modify.js";
export { default as modifyPath } from ".../src/modifyPath.js";
export { default as modulo } from ".../src/modulo.js";
export { default as move } from ".../src/move.js";
export { default as multiply } from ".../src/multiply.js";
export { default as nAry } from ".../src/nAry.js";
export { default as partialObject } from ".../src/partialObject.js";
export { default as negate } from ".../src/negate.js";
export { default as none } from ".../src/none.js";
export { default as not } from ".../src/not.js";
export { default as nth } from ".../src/nth.js";
export { default as nthArg } from ".../src/nthArg.js";
export { default as o } from ".../src/o.js";
export { default as objOf } from ".../src/objOf.js";
export { default as of } from ".../src/of.js";
export { default as omit } from ".../src/omit.js";
export { default as on } from ".../src/on.js";
export { default as once } from ".../src/once.js";
export { default as or } from ".../src/or.js";
export { default as otherwise } from ".../src/otherwise.js";
export { default as over } from ".../src/over.js";
export { default as pair } from ".../src/pair.js";
export { default as partial } from ".../src/partial.js";
export { default as partialRight } from ".../src/partialRight.js";
export { default as partition } from ".../src/partition.js";
export { default as path } from ".../src/path.js";
export { default as paths } from ".../src/paths.js";
export { default as pathEq } from ".../src/pathEq.js";
export { default as pathOr } from ".../src/pathOr.js";
export { default as pathSatisfies } from ".../src/pathSatisfies.js";
export { default as pick } from ".../src/pick.js";
export { default as pickAll } from ".../src/pickAll.js";
export { default as pickBy } from ".../src/pickBy.js";
export { default as pipe } from ".../src/pipe.js";
export { default as pipeWith } from ".../src/pipeWith.js";
export { default as pluck } from ".../src/pluck.js";
export { default as prepend } from ".../src/prepend.js";
export { default as product } from ".../src/product.js";
export { default as project } from ".../src/project.js";
export { default as promap } from ".../src/promap.js";
export { default as prop } from ".../src/prop.js";
export { default as propEq } from ".../src/propEq.js";
export { default as propIs } from ".../src/propIs.js";
export { default as propOr } from ".../src/propOr.js";
export { default as propSatisfies } from ".../src/propSatisfies.js";
export { default as props } from ".../src/props.js";
export { default as range } from ".../src/range.js";
export { default as reduce } from ".../src/reduce.js";
export { default as reduceBy } from ".../src/reduceBy.js";
export { default as reduceRight } from ".../src/reduceRight.js";
export { default as reduceWhile } from ".../src/reduceWhile.js";
export { default as reduced } from ".../src/reduced.js";
export { default as reject } from ".../src/reject.js";
export { default as remove } from ".../src/remove.js";
export { default as repeat } from ".../src/repeat.js";
export { default as replace } from ".../src/replace.js";
export { default as reverse } from ".../src/reverse.js";
export { default as scan } from ".../src/scan.js";
export { default as sequence } from ".../src/sequence.js";
export { default as set } from ".../src/set.js";
export { default as slice } from ".../src/slice.js";
export { default as sort } from ".../src/sort.js";
export { default as sortBy } from ".../src/sortBy.js";
export { default as sortWith } from ".../src/sortWith.js";
export { default as split } from ".../src/split.js";
export { default as splitAt } from ".../src/splitAt.js";
export { default as splitEvery } from ".../src/splitEvery.js";
export { default as splitWhen } from ".../src/splitWhen.js";
export { default as splitWhenever } from ".../src/splitWhenever.js";
export { default as startsWith } from ".../src/startsWith.js";
export { default as subtract } from ".../src/subtract.js";
export { default as sum } from ".../src/sum.js";
export { default as symmetricDifference } from ".../src/symmetricDifference.js";
export { default as symmetricDifferenceWith } from ".../src/symmetricDifferenceWith.js";
export { default as tail } from ".../src/tail.js";
export { default as take } from ".../src/take.js";
export { default as takeLast } from ".../src/takeLast.js";
export { default as takeLastWhile } from ".../src/takeLastWhile.js";
export { default as takeWhile } from ".../src/takeWhile.js";
export { default as tap } from ".../src/tap.js";
export { default as test } from ".../src/test.js";
export { default as andThen } from ".../src/andThen.js";
export { default as times } from ".../src/times.js";
export { default as toLower } from ".../src/toLower.js";
export { default as toPairs } from ".../src/toPairs.js";
export { default as toPairsIn } from ".../src/toPairsIn.js";
export { default as toString } from ".../src/toString.js";
export { default as toUpper } from ".../src/toUpper.js";
export { default as transduce } from ".../src/transduce.js";
export { default as transpose } from ".../src/transpose.js";
export { default as traverse } from ".../src/traverse.js";
export { default as trim } from ".../src/trim.js";
export { default as tryCatch } from ".../src/tryCatch.js";
export { default as type } from ".../src/type.js";
export { default as unapply } from ".../src/unapply.js";
export { default as unary } from ".../src/unary.js";
export { default as uncurryN } from ".../src/uncurryN.js";
export { default as unfold } from ".../src/unfold.js";
export { default as union } from ".../src/union.js";
export { default as unionWith } from ".../src/unionWith.js";
export { default as uniq } from ".../src/uniq.js";
export { default as uniqBy } from ".../src/uniqBy.js";
export { default as uniqWith } from ".../src/uniqWith.js";
export { default as unless } from ".../src/unless.js";
export { default as unnest } from ".../src/unnest.js";
export { default as until } from ".../src/until.js";
export { default as unwind } from ".../src/unwind.js";
export { default as update } from ".../src/update.js";
export { default as useWith } from ".../src/useWith.js";
export { default as values } from ".../src/values.js";
export { default as valuesIn } from ".../src/valuesIn.js";
export { default as view } from ".../src/view.js";
export { default as when } from ".../src/when.js";
export { default as where } from ".../src/where.js";
export { default as whereAny } from ".../src/whereAny.js";
export { default as whereEq } from ".../src/whereEq.js";
export { default as without } from ".../src/without.js";
export { default as xor } from ".../src/xor.js";
export { default as xprod } from ".../src/xprod.js";
export { default as zip } from ".../src/zip.js";
export { default as zipObj } from ".../src/zipObj.js";
export { default as zipWith } from ".../src/zipWith.js";
export { default as thunkify } from ".../src/thunkify.js";
